From b1a2f727ca99ecd3402d4b051b99cbfd24266e59 Mon Sep 17 00:00:00 2001
From: Eike Rathke <erack@redhat.com>
Date: Fri, 22 Jul 2022 22:17:11 +0200
Subject: Related: tdf#150011 Add HRK Croatian Kuna to Euro conversion wizard

Maybe just for completeness, it's removed from menu but might be
callable as macro.

Change-Id: Iade0be845186d3deb2f00f4aaa230c0b344cea72
Reviewed-on: https://gerrit.libreoffice.org/c/core/+/137372
Reviewed-by: Eike Rathke <erack@redhat.com>
Tested-by: Jenkins
---
 wizards/source/euro/Init.xba                        | 16 ++++++++++++++++
 wizards/source/resources/resources_en_US.properties |  1 +
 2 files changed, 17 insertions(+)

diff --git a/wizards/source/euro/Init.xba b/wizards/source/euro/Init.xba
index 623a0a53be46..9f56c503a347 100644
--- a/wizards/source/euro/Init.xba
+++ b/wizards/source/euro/Init.xba
@@ -89,6 +89,7 @@ Public sCurrSLOVAK as String
 Public sCurrESTONIAN as String
 Public sCurrLATVIAN as String
 Public sCurrLITHUANIAN as String
+Public sCurrCROATIAN as String
 
 Public sPrgsRETRIEVAL as String
 Public sPrgsCONVERTING as String
@@ -214,6 +215,7 @@ Dim LocWorkPath as String
 		sCurrESTONIAN = GetResText(&quot;CURRENCIES_16&quot;)
 		sCurrLATVIAN = GetResText(&quot;CURRENCIES_17&quot;)
 		sCurrLITHUANIAN = GetResText(&quot;CURRENCIES_18&quot;)
+		sCurrCROATIAN = GetResText(&quot;CURRENCIES_19&quot;)
 		.cmdCancel.Label =  sCANCEL
 		.cmdHelp.Label =  sHELP
 		.cmdBack.Label =  GetResText(&quot;STEP_ZERO_2&quot;)
@@ -393,6 +395,11 @@ Sub InitializeLanguages()
     LangIDValue(18,0,1) = &quot;LT&quot;
     LangIDValue(18,0,2) = &quot;-427&quot;
 
+&apos; CURRENCIES_CROATIAN
+    LangIDValue(19,0,0) = &quot;hr&quot;
+    LangIDValue(19,0,1) = &quot;HR&quot;
+    LangIDValue(19,0,2) = &quot;-41A&quot;
+
 End Sub
 
 
@@ -572,6 +579,15 @@ Dim i as Integer
 	CurrValue(18,4) = &quot;Lt&quot;
 	CurrValue(18,5) = &quot;LTL&quot;
 
+	CurrValue(19,0) = sCurrCROATIAN
+	&apos; real conversion rate
+	CurrValue(19,1) = 7.53450
+	&apos; rounded conversion rate
+	CurrValue(19,2) = 7.5
+	CurrValue(19,3) = &quot;kn&quot;
+	CurrValue(19,4) = &quot;kn&quot;
+	CurrValue(19,5) = &quot;HRK&quot;
+
 	i = -1
 	CurrSymbolList(0) = &quot;&quot;
 	CurrSymbolList(1) = &quot;&quot;
diff --git a/wizards/source/resources/resources_en_US.properties b/wizards/source/resources/resources_en_US.properties
index 32f9104e97e0..8649b2500e6a 100644
--- a/wizards/source/resources/resources_en_US.properties
+++ b/wizards/source/resources/resources_en_US.properties
@@ -448,6 +448,7 @@ CURRENCIES_15=Slovak Koruna
 CURRENCIES_16=Estonian Kroon
 CURRENCIES_17=Latvian Lats
 CURRENCIES_18=Lithuanian Litas
+CURRENCIES_19=Croatian Kuna
 STEP_LASTPAGE_0=Progress
 STEP_LASTPAGE_1=Retrieving the relevant documents...
 STEP_LASTPAGE_2=Converting the documents...
-- 
cgit v1.2.1

